testsuite/microprofile-tcks/pom.xml

```diff
@@ -69,6 +69,13 @@
              <skip>true</skip>
            </configuration>
          </plugin>
+         <plugin>
+           <groupId>org.sonatype.plugins</groupId>
+           <artifactId>nexus-staging-maven-plugin</artifactId>
+           <configuration>
+             <skip>true</skip>
+           </configuration>
+         </plugin>
```

Comment:
Should it be in the entire testsuite parent too? (I know this one doesn't inherit from testsuite parent.)

Reply:
Ah now I see the JIRA. Nevermind :-)